From: Kouei Abe <kouei.abe.cp@renesas.com>
Date: Fri, 20 Nov 2015 21:33:02 +0900
Subject: sata_rcar: Add compatible string for r8a7795
References: bsc#977572
Patch-Mainline: v4.5-rc1
Git-commit: fec7bc433a29ea75b83dd9447629c99b2d4d727a

R-Car H3 SoC has compatible SATA controller with R-Car Gen2 SoCs.

Signed-off-by: Kouei Abe <kouei.abe.cp@renesas.com>
Signed-off-by: Yoshihiro Kaneko <ykaneko0929@gmail.com>
Signed-off-by: Tejun Heo <tj@kernel.org>
Acked-by: Simon Horman <horms+renesas@verge.net.au>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 drivers/ata/sata_rcar.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/ata/sata_rcar.c b/drivers/ata/sata_rcar.c
index 8804127..2b20b31 100644
--- a/drivers/ata/sata_rcar.c
+++ b/drivers/ata/sata_rcar.c
@@ -854,6 +854,10 @@ static struct of_device_id sata_rcar_match[] = {
 		.compatible = "renesas,sata-r8a7793",
 		.data = (void *)RCAR_GEN2_SATA
 	},
+	{
+		.compatible = "renesas,sata-r8a7795",
+		.data = (void *)RCAR_GEN2_SATA
+	},
 	{ },
 };
 MODULE_DEVICE_TABLE(of, sata_rcar_match);
-- 
1.8.5.6

